### ISSUE TITLE
Install Error

### ISSUE THREAD

**Issue by mcguires5** [bethgelab/foolbox#407]
I cannot install foolbox with pip.
```
Running setup.py (path:C:\Users\Sean\AppData\Local\Temp\pip-install-ij3r22hf\foolbox\setup.py) egg_info for package foolbox
Running command python setup.py egg_info
running egg_info
creating C:\Users\Sean\AppData\Local\Temp\pip-install-ij3r22hf\foolbox\pip-egg-info\foolbox.egg-info
writing C:\Users\Sean\AppData\Local\Temp\pip-install-ij3r22hf\foolbox\pip-egg-info\foolbox.egg-info\PKG-INFO
writing dependency_links to C:\Users\Sean\AppData\Local\Temp\pip-install-ij3r22hf\foolbox\pip-egg-info\foolbox.egg-info\dependency_links.txt
writing requirements to C:\Users\Sean\AppData\Local\Temp\pip-install-ij3r22hf\foolbox\pip-egg-info\foolbox.egg-info\requires.txt
writing top-level names to C:\Users\Sean\AppData\Local\Temp\pip-install-ij3r22hf\foolbox\pip-egg-info\foolbox.egg-info\top_level.txt
writing manifest file 'C:\Users\Sean\AppData\Local\Temp\pip-install-ij3r22hf\foolbox\pip-egg-info\foolbox.egg-info\SOURCES.txt'
error: package directory 'foolbox\ext' does not exist

ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

**Comment by jonasrauber** [comment]
Can you try installing Foolbox 2.1 for now. Might be a bug I introduced in Foolbox 2.2.
`pip install foolbox==2.1.0`

What versions of `python`, `pip` and the `setuptools` package are you using?

**Comment by jonasrauber** [comment]
fixed in 2.2.1
